pom.xml

```diff
@@ -54,6 +55,11 @@
                 <artifactId>composer</artifactId>
                 <version>${composer.version}</version>
             </dependency>
+            <dependency>
```

Comment:
We don't want to rely on the integration module from tinkar core. We should just copy whatever code we think we'll need into this repo. But ideally, this should be minimal.